RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -272,6 +349,37 @@ class RNUnifiedContacts: NSObject {
 
     return emailAddresses
   }
+  
+  func transformNumber(number: NSDictionary) -> CNLabeledValue {
+    var label = String()
+    if (number["label"] as! String == "home") {
```

Comment:
I mentioned in a previous comment that exporting Constants for this would be better, as per: https://facebook.github.io/react-native/docs/native-modules-ios.html#exporting-constants

Also, using a `Switch` might be a better option here to avoid code duplication. i.e. `number["label"] as! String`

Also, with this approach, `label` could be a blank string. Two ways of handling this: 

1. Default to something, "home" perhaps. 
2. Throw an exception. 

What do you think?